resources/public/api/conf/1.0/retrieve.yaml

```diff
@@ -3,38 +3,14 @@ description: |
   number and tax year must be provided.
   
   ### Test data
-  <p>Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.</p>
-  <table>
-    <thead>
-        <tr>
-            <th>Header Value (Gov-Test-Scenario)</th>
-            <th>Scenario</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulates success response.</p></td>
-        </tr>
-        <tr>
-            <td><p>NOT_FOUND</p></td>
-            <td><p>Simulates the scenario where no data is found.</p></td>
-        </tr>
-        <tr>
-            <td><p>STATEFUL</p></td>
-            <td><p>Performs a stateful retrieve.</p></td>
-        </tr>
-        <tr>
-            <td><p>DYNAMIC</p></td>
-            <td><p>The following response values will change to correspond to the values submitted in the request:
-              <ul>
-                  <li>submittedOn</li>
-                  <li>taxYear</li>
-              </ul>
-            </p></td>
-        </tr>
-    </tbody>
-  </table>
+  Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario)      | Scenario                                                                                                                               |
+  |---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                         | Simulates success response.                                                                                                            |
+  | NOT_FOUND                             | Simulates the scenario where no data is found.                                                                                         |                                                 
+  | STATEFUL                              | Performs a stateful retrieve.                                                                                                          |
+  | DYNAMIC                               | The following response values will change to correspond to the values submitted in the request: <br> <br> • submittedOn <br> • taxYear |
```

Comment:
I think we only really need the second an third `<br>`. The extra empty line looks a little strange